docs/quickstart_guide.md

```diff
@@ -25,6 +25,9 @@ To see how to setup a single node cluster with stock-only or gVisor, see [Develo
     1. [Deploy Functions](#1-deploy-functions)
```

Comment:
please run this manual by one of your DSL colleagues as well. Ask them to try it out and report their experience here in this PR, apply their feedback if any.

Reply:
I had Jonas and Estella look over as well. They helped improve the quickstart guide steps for stargz but encountered no problems running the platform with stargz.